FieldDisplay.java

```diff
@@ -14,7 +14,9 @@ public FieldDisplay(BattleField field, boolean showShips) {
 
     public void render() {
         int size = mField.size();
+        System.out.println("  1 2 3 4 5 6 7 8 9 10");
```

Comment:
don't print out the numbers directly, you should loop through them instead.
If your field size changes this won't be accurate.